docker/env/Dockerfile

```diff
@@ -4,7 +4,8 @@ ADD install-prereqs.sh install-prereqs.sh
 RUN ./install-prereqs.sh docker
 RUN yum update -y \
     && yum install -y https://centos7.iuscommunity.org/ius-release.rpm \
-    && yum install -y python36u python36u-libs python36u-devel python36u-pip
+    && yum install -y python36u python36u-libs python36u-devel python36u-pip \
+    && yum install -y iproute
```

Comment:
I think it would be better to add it in:
https://github.com/scylladb/scylla-cluster-tests/blob/master/install-prereqs.sh

And not mix it with the python install layer of the docker

Reply:
Also you need to update the `docker/env/version`, cause with updating it. a new docker won't be built.

Reply:
> I think it would be better to add it in:
> https://github.com/scylladb/scylla-cluster-tests/blob/master/install-prereqs.sh
> 
> And not mix it with the python install layer of the docker

It has docker installation in it, not sure if it is really needed there.

Reply:
> Also you need to update the `docker/env/version`, cause with updating it. a new docker won't be built.

Done.

Reply:
> > I think it would be better to add it in:
> > https://github.com/scylladb/scylla-cluster-tests/blob/master/install-prereqs.sh
> > And not mix it with the python install layer of the docker
> 
> It has docker installation in it, not sure if it is really needed there.

The docker installation is needed since we user docker within hydra, a.k.a docker-in-docker.
For the docker backend support, but also for rsyslogd and autossh. and hydra show-monitor command.